packages/openactive-broker-microservice/app.js

```diff
@@ -2,7 +2,7 @@
 const express = require('express');
 const logger = require('morgan');
 const request = require('request');
-const axios = require('axios');
+const { default: axios } = require('axios');
```

Comment:
This is unfortunately required for axios types to be registered correctly due to the way that they transpile ES Modules to CommonJS